### PR TITLE
Add IGB FAQ to the GTN

### PR DESCRIPTION
**Pull request by paige-kulzer**
Added an FAQ for visualizing Galaxy History files in the Integrated Genome Browser.

### CODE REVIEW

**Comment by bgruening** [comment]
Please add yourself to the Contributors file: https://github.com/galaxyproject/training-material/blob/main/CONTRIBUTORS.yaml

:-)

**Comment by paige-kulzer** [comment]
Updated the Contributors file with my name and added that change to this PR accordingly!

File: CONTRIBUTORS.yaml [hunk 1]
```diff
@@ -1889,6 +1889,21 @@ pcm32:
...
+pkulzerUNC:
+  name: Paige Kulzer
+  joined: 2024-10
+  email: pkulzer@charlotte.edu
+  linkedin: paige-kulzer-210b97274
+  contact_for_training: true
+  bio:
+    "My current role involves bioinformatics, data management, and training for a genome browser called the Integrated Genome Browser."
+  affiliations:
+    - University of North Carolina at Charlotte
```
Comment (hexylena) [inline CONTRIBUTORS.yaml:1901]: affiliations have to be referenced in the `ORGANISATIONS.yaml` file.

**Review by hexylena — COMMENTED** [review]
Welcome! Congrats on the first contribution it looks good!

File: CONTRIBUTORS.yaml [hunk 2]
```diff
@@ -1889,6 +1889,21 @@ pcm32:
     name: Pablo Moreno
     joined: 2019-06
 
+pkulzerUNC:
```
Comment (hexylena) [inline CONTRIBUTORS.yaml:1892]:
```suggestion
paige-kulzer:
```

the value here and in the FAQ must match

**Comment by paige-kulzer** [comment]
I've removed mention of affiliation for now and corrected the name in the Contributors file. Thanks for those catches!

File: faqs/galaxy/visualisations_igb.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+You can open some file types in Integrated Genome Browser (IGB), a desktop genome browser. ([Supported File Types](https://wiki.bioviz.org/confluence/display/igbman/File+Formats))
+
+Here's how:
+
+1. Install IGB on your computer ([download page](https://bioviz.org/download.html)).
+2. Start IGB.
+3. In Galaxy, click the desired dataset's name to expand it.
+   - Check that the reference genome (dbkey) is set
+   ([instructions](https://training.galaxyproject.org/training-material/faqs/galaxy/datasets_change_dbkey.html))
```
Comment (bgruening) [inline faqs/galaxy/visualisations_igb.md:17]: please use the link function for internal links, like here for example: https://github.com/galaxyproject/training-material/blob/c4ad059255fb949d6cf861d96de87858743016ea/faqs/gtn/gtn_event_create.md?plain=1#L10

**Review by bgruening — APPROVED** [review]
Thanks @paige-kulzer!

**Pull request merged**